client/flutter/lib/carousel_page.dart

```diff
@@ -59,9 +91,7 @@ class CarouselView extends StatelessWidget {
           ),
           Align(
             alignment: FractionalOffset.bottomCenter,
-            child: Container(
-                padding: EdgeInsets.only(bottom: 20),
-                child: pageViewIndicator(context)),
+            child: Container(padding: EdgeInsets.only(bottom: 20), child: pageViewIndicator(context)),
```

Comment:
is there a preferred code style on this from `flutter format`? prefer to standardize once and stick with it then have changes in a PR to review that aren't related to the overall purpose of the PR. not a blocker on this PR, just a general comment...

Reply:
One style note on this, both usage of `EdgeInsets` above use `const` and use `10.0` but this uses the integer notation and no const. I don't know which one is preferable but we should probably be consistent.

Reply:
I looked for a `CONTRIBUTING.md` and in the Wiki to see, but guidance doesn't appear to exist, so I'm not sure what the coding conventions should be. My suggestion is to define it as the output of `flutter format -l 100 .` and leave it at that. Easily enforceable, eliminates all the time and effort talking about and fixing formatting, and it does a pretty good job, even if it's not perfect.

Reply:
If you would like to easily enforce but eliminate all the time and effort talking about formatting once-in-for-all 😆I would strongly recommend sticking to the [style guide for Flutter's own repo](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#formatting). For example: preferred maximum line length is 80.

Would be greatly beneficial to all new contributors even if we just link the above official Flutter repo's style guide in `CONTRIBUTING.md` so we are all in the same page. I spent some time looking for one before I started to code but couldn't find anything.